File: first_page_module/src/main/java/com/phone/first_page_module/FirstPageServiceImpl.java
package com.phone.first_page_module;

import android.content.Context;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.phone.common_library.service.FirstPageService;
import com.phone.common_library.bean.FirstPageResponse;

import java.util.List;

@Route(path = "/first_page_module/FirstPageServiceImpl")
public class FirstPageServiceImpl implements FirstPageService {

    private static final String TAG = FirstPageServiceImpl.class.getSimpleName();

    @Override
    public void setFirstPageDataList(List<FirstPageResponse.ResultData.JuheNewsBean> firstPageDataList) {
        this.firstPageDataList.clear();
        this.firstPageDataList.addAll(firstPageDataList);
    }

    @Override
    public List<FirstPageResponse.ResultData.JuheNewsBean> getFirstPageDataList() {
        return firstPageDataList;
    }

    @Override
    public void init(Context context) {

    }
}

File: first_page_module/src/main/java/com/phone/first_page_module/FirstPageServiceImpl.kt
package com.phone.first_page_module

import android.content.Context
import com.alibaba.android.arouter.facade.annotation.Route
import com.phone.common_library.bean.FirstPageResponse.ResultData.JuheNewsBean
import com.phone.common_library.service.IFirstPageService

@Route(path = "/first_page_module/FirstPageServiceImpl")
class FirstPageServiceImpl : IFirstPageService {

    private val TAG = FirstPageServiceImpl::class.java.simpleName

    override fun setFirstPageDataList(firstPageDataList: MutableList<JuheNewsBean?>) {
        mFirstPageDataList.clear()
        mFirstPageDataList.addAll(firstPageDataList)
    }

    override fun getFirstPageDataList(): MutableList<JuheNewsBean?> {
        return mFirstPageDataList
    }

    override fun init(context: Context?) {
    }

}